### DISCUSSION TITLE
Yellow beat bars getting very distracting

### DISCUSSION THREAD

**Discussion by coiledmocha** [ppy/osu#19608] [Q&A] [answered]
What are these weird lines and how do I turn them off. Very distracting, and doens't seem normal. Thanks in advance.

![osumaniaweirdlines](https://user-images.githubusercontent.com/68021563/183261153-063349bf-ca91-4531-9df3-ca4da1c64776.jpg)

**Comment by nagi-desuuu** [comment, marked as answer]
These lines are bar lines that visually help you get a grasp on the song's timing.
Currently, there is no option to turn it off.

**Reply by coiledmocha** [reply]
Thanks. I'll just leave the song out till theres an option to turn it off.

**Reply by peppy** [reply]
We're very aware that these are too present and ugly, and it will be addressed in the near future.
